### PR TITLE
Require 2.452.4

### PR DESCRIPTION
**Pull request by jonesbusy**
Hello `discard-old-build` developers!

This is an automated pull request created by the [Jenkins Plugin Modernizer](https://github.com/jenkins-infra/plugin-modernizer-tool) tool. The tool has applied the following recipes to modernize the plugin:
<details>
    <summary>Upgrade to latest recommended core version and ensure the bom is matching the core version</summary>
    <p><em>io.jenkins.tools.pluginmodernizer.UpgradeToRecommendCoreVersion</em></p>
    <blockquote>Upgrade to latest recommended core version and ensure the bom is matching the core version.</blockquote>
</details>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,7 +87,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-${jenkins.baseline}.x</artifactId>
-        <version>2102.v854b_fec19c92</version>
+        <version>3867.v17b_41ce6fcb_c</version>
```
Comment (MarkEWaite) [inline pom.xml:90]: Needs to use the released version, not the incremental version of the plugin BOM.
```suggestion
        <version>3850.vb_c5319efa_e29</version>
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
The plugin BOM version needs to be a released version rather than an incremental version.  The incremental version has not completed the full testing process that is used for released versions.

File: pom.xml (hunk 1 above)
Comment (jonesbusy) [reply pom.xml:90]: Good catch. Not sure how we go this version

Comment (jonesbusy) [reply pom.xml:90]: https://github.com/jenkins-infra/plugin-modernizer-tool/issues/534

**Pull request closed without merging**